assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js

```diff
@@ -0,0 +1,215 @@
+			} )
+	);
+
+	const dateRangeLength = useSelect( ( select ) =>
+		select( CORE_USER ).getDateRangeNumberOfDays()
+	);
+
+	const reportArgs = {
+		startDate,
+		endDate,
+		compareStartDate,
+		compareEndDate,
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+		dimensions: [ 'ga:date' ],
+	};
+
+	const data = useInViewSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getReport( reportArgs )
+	);
+
+	const loading = useSelect(
+		( select ) =>
+			! select( MODULES_ANALYTICS ).hasFinishedResolution( 'getReport', [
+				reportArgs,
+			] )
+	);
+
+	const error = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getErrorForSelector( 'getReport', [
+			reportArgs,
+		] )
+	);
+
+	if ( ! ( analyticsModuleActive && analyticsModuleConnected ) ) {
+		return null;
+	}
+
+	if ( error ) {
+		return <WidgetReportError moduleSlug="analytics" error={ error } />;
+	}
+
+	const googleChartData =
+		extractAnalyticsDashboardData(
+			data || [],
+			0,
+			dateRangeLength,
+			0,
+			1,
+			[ __( 'Unique Visitors', 'google-site-kit' ) ],
+			[ ( x ) => parseFloat( x ).toLocaleString() ]
+		) || [];
+
+	const dates = googleChartData.slice( 1 ).map( ( [ date ] ) => date );
+	const options = {
+		...WPDashboardUniqueVisitorsChartWidget.chartOptions,
+		hAxis: {
+			...WPDashboardUniqueVisitorsChartWidget.chartOptions.hAxis,
+			ticks: [ dates[ 0 ], dates[ dates.length - 1 ] ],
+		},
+		series: {
+			0: {
+				color: WPDashboardUniqueVisitorsChartWidget.statsColor,
+				targetAxisIndex: 0,
+			},
+			1: {
+				color: WPDashboardUniqueVisitorsChartWidget.statsColor,
+				targetAxisIndex: 0,
+				lineDashStyle: [ 3, 3 ],
+				lineWidth: 1,
+			},
+		},
+	};
+
+	const currentValueIndex = 2;
+	const previousValueIndex = 3;
+	const isZeroChart = ! googleChartData
+		.slice( 1 )
+		.some(
+			( datum ) =>
+				datum[ currentValueIndex ] > 0 ||
+				datum[ previousValueIndex ] > 0
+		);
+
+	if ( isZeroChart ) {
+		options.hAxis.ticks = [ new Date() ];
+	}
+
+	return (
+		<div className="googlesitekit-unique-visitors-chart-widget">
+			<h3>Unique Visitors from Search</h3>
+			<GoogleChart
+				chartType="LineChart"
+				data={ googleChartData }
+				loadingHeight="270px"
+				loadingWidth="100%"
+				loaded={ ! ( loading || isGatheringData === undefined ) }
+				options={ options }
+				gatheringData={ isGatheringData }
+			/>
+		</div>
+	);
+};
+
+WPDashboardUniqueVisitorsChartWidget.chartOptions = {
+	animation: {
+		startup: true,
+	},
+	chart: {
+		title: 'Unique Visitors from Search',
+	},
+	statsColor: '#5c9271',
+	curveType: 'function',
+	height: 270,
+	width: '100%',
+	chartArea: {
+		height: '80%',
+		left: 20,
+		right: 20,
+	},
+	legend: {
+		position: 'top',
+		textStyle: {
+			color: '#616161',
+			fontSize: 12,
+		},
+	},
+	hAxis: {
+		format: 'M/d/yy',
+		gridlines: {
+			color: '#fff',
+		},
+		textStyle: {
+			color: '#616161',
+			fontSize: 12,
+		},
+	},
+	vAxis: {
+		textPosition: 'none',
```

Comment:
Please restore the `gridlines.color` setting as this is still needed for the horizontal grid lines.

```js
		gridlines: {
			color: '#eee',
		},
```